snakedeploy/exceptions.py

```diff
@@ -31,3 +31,7 @@ class UnrecognizedProviderError(ValueError):
 
     def __init__(self, message, *args, **kwargs):
         super(UnrecognizedProviderError, self).__init__(*args, **kwargs)
+
+
+class UserError(Exception):
```

Comment:
Is the base Exception class the right one to use here (e.g., and not ValueError if it's typically something that the user did?"

Reply:
I think the base exception should be ok, but if you prefer a ValueError that would be fine for me as well. Your choice.

Reply:
okay let's do ValueError.